test/CometFlexVotingFork.t.sol

```diff
@@ -0,0 +1,1036 @@
+      0x514910771AF9Ca656af840dff83E8264EcF986CA, // asset, LINK
+      0x2c1d072e956AFFC0D435Cb7AC38EF18d24d9127c, // priceFeed
+      18, // decimals
+      790_000_000_000_000_000, // borrowCollateralFactor
+      850_000_000_000_000_000, // liquidateCollateralFactor
+      930_000_000_000_000_000, // liquidationFactor
+      1_250_000_000_000_000_000_000_000 // supplyCap
+    );
+    Configuration memory _config = Configuration(
+      COMPOUND_GOVERNOR,
+      0xbbf3f1421D886E9b2c5D716B5192aC998af2012c, // pauseGuardian
+      address(govToken), // baseToken
+      0x8fFfFfd4AfB6115b954Bd326cbe7B4BA576818f6, // baseTokenPriceFeed, using the chainlink
+        // USDC/USD price feed
+      0x285617313887d43256F852cAE0Ee4de4b68D45B0, // extensionDelegate
+      800_000_000_000_000_000, // supplyKink
+      1_030_568_239 * 60 * 60 * 24 * 365, // supplyPerYearInterestRateSlopeLow
+      12_683_916_793 * 60 * 60 * 24 * 365, // supplyPerYearInterestRateSlopeHigh
+      0, // supplyPerYearInterestRateBase
+      800_000_000_000_000_000, // borrowKink
+      1_109_842_719 * 60 * 60 * 24 * 365, // borrowPerYearInterestRateSlopeLow
+      7_927_447_995 * 60 * 60 * 24 * 365, // borrowPerYearInterestRateSlopeHigh
+      475_646_879 * 60 * 60 * 24 * 365, // borrowPerYearInterestRateBase
+      600_000_000_000_000_000, // storeFrontPriceFactor
+      1_000_000_000_000_000, // trackingIndexScale
+      0, // baseTrackingSupplySpeed
+      3_257_060_185_185, // baseTrackingBorrowSpeed
+      1_000_000_000_000, // baseMinForRewards
+      100_000_000, // baseBorrowMin
+      5_000_000_000_000, // targetReserves
+      _assetConfigs
+    );
+
+    cToken = new CometFlexVoting(_config, address(flexVotingGovernor));
+
+    cToken.initializeStorage();
+    // ========= END DEPLOY NEW COMET ========================
+
+    // TODO is there anything we need to do to make this an "official" Comet?
+  }
+
+  // ------------------
+  // Helper functions
+  // ------------------
+
+  function _mintGovAndSupplyToCompound(address _who, uint256 _govAmount) internal {
+    govToken.exposed_mint(_who, _govAmount);
+    vm.startPrank(_who);
+    govToken.approve(address(cToken), type(uint256).max);
+    cToken.supply(address(govToken), _govAmount);
+    vm.stopPrank();
+  }
+
+  function _createAndSubmitProposal() internal returns (uint256 proposalId) {
+    // Proposal will underflow if we're on the zero block.
+    if (block.number == 0) vm.roll(42);
+
+    // Create a dummy proposal.
+    bytes memory receiverCallData = abi.encodeWithSignature("mockReceiverFunction()");
+    address[] memory targets = new address[](1);
+    uint256[] memory values = new uint256[](1);
+    bytes[] memory calldatas = new bytes[](1);
+    targets[0] = address(receiver);
+    values[0] = 0; // no ETH will be sent
+    calldatas[0] = receiverCallData;
+
+    // Submit the proposal.
+    proposalId = flexVotingGovernor.propose(targets, values, calldatas, "A great proposal");
```

Comment:
> Or is the intent of this method that the test contract is always proposing?

Correct, the test contract is always proposing. This works because the proposal threshold is [0 by default](https://github.com/OpenZeppelin/openzeppelin-contracts/blob/96b95592c300a1b38953b276f0b6c77017f5c942/contracts/governance/Governor.sol#L199-L204), so the proposer doesn't need to have any govTokens

I'll add a comment about this. I agree it's not obvious

Reply:
Done in 56bf3cf, thanks